fe/fe-core/src/test/resources/conf/config_test.properties

```diff
@@ -0,0 +1 @@
+tablet_sched_slot_num_per_path = 3
```

Comment:
what's the purpose of the three properites?

Reply:
They are loaded in the ut to test some behaviors of ConfigBase class.